src/utils/contractHelpers.ts

```diff
@@ -127,9 +128,11 @@ export const getEasterNftContract = (signer?: ethers.Signer | ethers.providers.P
 export const getCakeVaultContract = (signer?: ethers.Signer | ethers.providers.Provider) => {
   return getContract(cakeVaultAbi, getCakeVaultAddress(), signer)
 }
+
 export const getPredictionsContract = (signer?: ethers.Signer | ethers.providers.Provider) => {
-  return getContract(predictionsAbi, getPredictionsAddress(), signer)
+  return getContract(predictionsAbi, getPredictionsAddress(), signer) as PredictionsContract
```

Comment:
Tried to use a generic on `useContract` but it threw an error that I could not figure out how to resolve.